scripted-tests/run/java-io-file-output-stream/build.sbt

```diff
@@ -0,0 +1,49 @@
+import java.io.File
+import java.io.FileOutputStream
+import java.io.FileInputStream
+
+ScalaNativePlugin.projectSettings
```

Comment:
Here and in the other new scripted tests, we should do `enablePlugins(ScalaNativePlugin)` instead. This has changed recently in #621.